deployments/helm/nvidia-device-plugin/templates/daemonset-gfd.yml

```diff
@@ -199,6 +201,10 @@ spec:
         - name: host-sys
           hostPath:
             path: "/sys"
+        - name: nvidia-imex-dir
+          type: DirectoryOrCreate
+          hostPath:
+            path: {{ include "nvidia-device-plugin.filepathJoin" (list "/" .Values.nvidiaDriverRoot "etc/nvidia-imex") }}
```

Comment:
```suggestion
            path: {{ include "nvidia-device-plugin.filepathJoin" (list "/" .Values.nvidiaDriverRoot "/etc/nvidia-imex") }}
```

Reply:
is not needed if `.Values.nvidiaDriverRoot` is null the func will handle

Reply:
i have tested `.Values.nvidiaDriverRoot` =

- `null`
- /proc/nvidia
- /proc/nvidia/
- proc/nvidia

Reply:
Its not about being needed. Its about how you normally pass paths to a join function. I think we want it for clarity.

Reply:
ok, for clarity I take it, added.